easysid.cpp

```diff
@@ -16,6 +16,7 @@
 SID sid;
 PcmOutput *pcm = NULL;
 PcmOutput *wav = NULL;
+int sample_rate = 44100;
```

Comment:
I suggest using a global-style name for this, such as `g_sample_rate`. Otherwise it just looks like the name of any other variable.